package/autoyast2.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Wed Apr  1 10:51:35 UTC 2020 - Stasiek Michalski <stasiek@michalski.cc>
+
+- Restore some missing icons (related to bsc#1168123, boo#1109310).
```

Comment:
Maybe you could include the also the new bug report

https://bugzilla.opensuse.org/show_bug.cgi?id=1168281

Just to keep the reference.

Anyway, it looks good to me as it is.